build.sbt

```diff
@@ -38,20 +38,28 @@ lazy val root = (project in file("."))
     UglifyKeys.compressOptions := Seq("unused=false", "dead_code=false"),
     pipelineStages := Seq(digest),
     // below line required to force asset pipeline to operate in dev rather than only prod
-    pipelineStages in Assets := Seq(concat,uglify),
+    pipelineStages in Assets := Seq(concat, uglify),
     // only compress files generated by concat
     includeFilter in uglify := GlobFilter("bindingtarifftraderfrontend-*.js")
   )
   .settings(inConfig(TemplateTest)(Defaults.testSettings): _*)
   .settings(
-    unmanagedSourceDirectories in Test := Seq((baseDirectory in Test).value / "test"),
+    unmanagedSourceDirectories in Test := Seq(
+      (baseDirectory in Test).value / "test/unit",
+      (baseDirectory in Test).value / "test/util",
+      (baseDirectory in Test).value / "test/resources"
+    ),
     addTestReportOption(Test, "test-reports")
   )
   .configs(IntegrationTest)
   .settings(inConfig(TemplateItTest)(Defaults.itSettings): _*)
   .settings(
     Keys.fork in IntegrationTest := false,
-    unmanagedSourceDirectories in IntegrationTest := Seq((baseDirectory in IntegrationTest).value / "it"),
+    unmanagedSourceDirectories in IntegrationTest := Seq(
+      (baseDirectory in IntegrationTest).value / "test/it",
+      (baseDirectory in IntegrationTest).value / "test/util",
+      (baseDirectory in IntegrationTest).value / "test/resources"
```

Comment:
This is not a source directory, but a resource directory.
See how it has been done in `binding-tariff-filestore`.